assets/components/NavBar/Button.css

```diff
@@ -0,0 +1,46 @@
+:root{
```

Comment:
We are using SCSS, for CSS pre-processing. Replace this file with a SCSS pre-processor file in the /styles directory tree matching the structure of the element the .scss file is for, look at other components and their styles files for comparison.